src/cascade/input_data/db/bundle.py

```diff
@@ -196,7 +196,7 @@ def _upload_bundle_data_to_tier_3(cursor, model_version_id, bundle_data):
     )
     """
 
-    cursor.executemany(insert_query, bundle_data.values())
+    cursor.executemany(insert_query, bundle_data.values)
```

Comment:
Looks like df.to_sql needs: 
    con : sqlalchemy.engine.Engine or sqlite3.Connection: 
rather than:
    db_tools.ezfuncs.get_connection(conn_def, raw=False, connectable=True)
        Returns: | sqlalchemy.pool._ConnectionFairy or db_tools.mysqlapis.MySQLConnection:
I made a quick attempt at calling ezfuncs.get_engine() instead of ezfuncs.get_connection(), and I got an error that user dbview is not allowed to insert into table t3_model_version_csmr


with a cursor, I unsuccessfully tried each of these individually as inputs to executemany:
    csmr_data_values = list(csmr_data.itertuples(index=False))
    csmr_data_values = [tuple(x) for x in csmr_data.values] 
    For this last one, it doesn't like the numpy types, float or int., 
    eg., AttributeError: 'numpy.int64' object has no attribute 'translate'

Reply:
PR #14 has the bundle code fiddles for this one, as you know, just saying